core/src/main/scala/kafka/controller/KafkaController.scala

```diff
@@ -1108,8 +1108,6 @@ class KafkaController(val config: KafkaConfig,
         (tp, controllerContext.partitionReplicaAssignment(tp) )
       }.toMap.groupBy { case (_, assignedReplicas) => assignedReplicas.head }
 
-    debug(s"Preferred replicas by broker $preferredReplicasForTopicsByBrokers")
```

Comment:
Maybe, instead of removing it, changing it to `trace`? what do you think?

Reply:
I'd rather just get rid of it. Traditionally kafka tells users to run with TRACE logging enabled for the controller so changing the level does not reduce the noise in the common case. We need to break the habit of logging messages which include all partitions in the cluster.